o-markitdigital-equity-highlight/main.scss

```diff
@@ -1,12 +1,20 @@
 // Output grid helper classes and data-attributes
+
+$o-typography-is-silent: false;
+$o-fonts-is-silent: false;
+$o-colors-is-silent: false;
```

Comment:
The same applies to all of the other things you're turning off silent mode for. To use the mixins (like `oColorsFor(page)` you don't need to turn off silent mode, you just need to import it as you're doing below.